### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by lexesjan**
## Purpose

Anime episode detection does not work (happened during mirgration to redux) when the anime is correctly identified using MALSync (for example on 9anime). We are unable to change Firefox keybinds and the extension size is bloated.

## Proposed Change

Fix anime episode detection and keybind bug and analyse the webpack bundle for optimisation.

## Checklist

- [x] Switch to babel for better tree-shaking (~2mb zipped to ~500kb zipped)
- [x] Fix anime episode detection bug
- [x] Fix Firefox keybind bug
- [x] Bump version patch

## Additional

Out of scope changes:
- [x] Use web-ext-webpack-plugin for extension reloading
- [x] General code cleanup
- [x] Add keyboard selection for anime search
- [x] Clean up yarn scripts
- [x] Update dependencies

### CODE REVIEW

File: src/api/aniskip-http-client/aniskip-http-client.ts [hunk 1]
```diff
@@ -24,6 +24,7 @@ export class AniskipHttpClient extends BaseHttpClient {
    * @param animeId MAL id to get the skip times of.
    * @param episodeNumber Episode number of the anime to get the skip times of.
    * @param type Type of skip times to get.
+   * @param episodeLength Length of the episode.
```
Comment (dabreadman) [inline src/api/aniskip-http-client/aniskip-http-client.ts:27]: lul
image

Comment (lexesjan) [reply src/api/aniskip-http-client/aniskip-http-client.ts:27]: lmaoo

**Review by dabreadman — APPROVED** [review]
Pray to 
image

**Comment by lexesjan** [comment]
> Pray to image

🙏🙏🙏

**Pull request merged**